lib/database/models/index.js

```diff
@@ -60,6 +61,7 @@ module.exports = (sequelize) => {
         Subsystem: Subsystem(sequelize),
         Tag: Tag(sequelize),
         User: User(sequelize),
+        LHCPeriod: LHCPeriod(sequelize),
```

Comment:
We don't use full capital, `LhcPeriod` (dumb or not we can discuss, but for now it's the convention)

Reply:
renamed